static/src/javascripts/projects/common/modules/identity/sign-in-gate/types.js

```diff
@@ -10,7 +10,7 @@ export type ComponentEventParams = {
     componentId?: string,
     abTestName: string,
     abTestVariant: string,
-    viewId?: string,
```

Comment:
Why was this made optional in the original PR? Hard to parse.

Reply:
i have a feeling that this was a mistake on my part when initially setting up the sign in gate, since this parameter is required in identity api to fire the component event